libbeat/outputs/otelconsumer/otelconsumer.go

```diff
@@ -103,30 +102,6 @@ func (out *otelConsumer) logsPublish(ctx context.Context, batch publisher.Batch)
 
 	err := out.logsConsumer.ConsumeLogs(ctx, pLogs)
 	if err != nil {
-		// If the batch is too large, the elasticsearchexporter will
-		// return a 413 error.
-		//
-		// At the moment, the exporter does not support batch splitting
-		// on error so we do it here.
-		//
-		// See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36163.
-		if strings.Contains(err.Error(), "Request Entity Too Large") {
-			// Try and split the batch into smaller batches and retry
-			if batch.SplitRetry() {
-				st.BatchSplit()
-				st.RetryableErrors(len(events))
-			} else {
-				// If the batch could not be split, there is no option left but
-				// to drop it and log the error state.
-				batch.Drop()
-				st.PermanentErrors(len(events))
-				out.log.Errorf("the batch is too large to be sent: %v", err)
-			}
```

Comment:
I wonder if it makes sense to split and retry anyway if we get a transient error (else branch of consumererror.IsPermanent(err) below). It might be a safer approach.